### PR TITLE
Storybook: Mock MatchMedia

### PR DESCRIPTION
**Pull request by timhaines**
jsdom reportedly doesn't support [window.matchMedia](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia).  This PR adds a mock for it when loading stories.

### CODE REVIEW

**Comment by timhaines** [comment]
Travis is currently failing on node_js 4 as the new react-match-media package introduced in integration tests requires node 5+

**Comment by timhaines** [comment]
@fotinakis Lovely PR for you to review when you get a moment.  🌱 

Stretching the limits of my knowledge with the bash scripting here.  Let me know if you know a better way than what I added in the travis.yml file.

File: .travis.yml [hunk 1]
```diff
@@ -10,6 +10,9 @@ matrix:
       env: PERCY_ENABLE=0
 script:
   - npm test
-  - cd packages/react-percy-storybook && yarn link
-  - cd ../../integration-tests && yarn && yarn link @percy-io/react-percy-storybook
-  - npm run storybook:percy
+  - >
```
Comment (fotinakis) [inline .travis.yml]: Nice. With this amount of directory state stuff you could potentially throw this into a file like `utils/integration-tests.sh` and just run that here.

Comment (timhaines) [inline .travis.yml]: @fotinakis Thanks, have put this into scripts/run-integration-test-snapshots.sh.

**Pull request merged**